cmd/livepeer/starter/starter.go

```diff
@@ -1365,17 +1309,24 @@ func StartLivepeer(ctx context.Context, cfg LivepeerConfig) {
 
 	}()
 
-	if n.NodeType == core.TranscoderNode {
+	if n.NodeType == core.TranscoderNode || n.NodeType == core.AIWorkerNode {
 		if n.OrchSecret == "" {
 			glog.Exit("Missing -orchSecret")
 		}
 		if len(orchURLs) <= 0 {
 			glog.Exit("Missing -orchAddr")
 		}
-
+	}
+	if n.NodeType == core.TranscoderNode {
+		//start the transcoder
 		go server.RunTranscoder(n, orchURLs[0].Host, core.MaxSessions, transcoderCaps)
 	}
 
+	if n.AIWorker != nil {
```

Comment:
For consistency with L1320 would this make sense?

```
if n.NodeType == core.AIWorkerNode { ... }
```

since you'd only want to call `server.RunAIWorker()` when the node is a standalone aiworker (and not when the node is a combined orchestrator + aiworker).

Reply:
agree, updated